src/views/cart.ejs

```diff
@@ -0,0 +1,198 @@
+<%- include('./navbar_v2') %>
+    <section class="h-100 gradient-custom">
+        <div class="container py-5">
+          <div class="row d-flex justify-content-center my-4">
+            <div class="col-md-8">
+              <div class="card mb-4">
+                <div class="card-header py-3">
+                  <h5 class="mb-0">Cart - 2 items</h5>
+                </div>
+                <div class="card-body">
+
+                  <!-- Single item -->
+                  <div class="row">
```

Comment:
Div z pojedynczym produktem też do osobnego `.ejs`.

Reply:
Masz racje. Będzie to dodane w następnym etapie.